src/gui/folderman.cpp

```diff
@@ -403,12 +415,12 @@ void FolderMan::backwardMigrationSettingsKeys(QStringList *deleteKeys, QStringLi
 
     auto processSubgroup = [&](const QString &name) {
         settings->beginGroup(name);
-        const int foldersVersion = settings->value(QLatin1String(versionC), 1).toInt();
+        const int foldersVersion = settings->value(versionC(), 1).toInt();
         if (foldersVersion <= maxFoldersVersion) {
             const auto &childGroups = settings->childGroups();
             for (const auto &folderAlias : childGroups) {
                 settings->beginGroup(folderAlias);
-                const int folderVersion = settings->value(QLatin1String(versionC), 1).toInt();
+                const int folderVersion = settings->value(versionC(), 1).toInt();
```

Comment:
`1` is another magic number. Can you make constants somewhere in 1 place (in a header file?) and add some comments to them? I've now seen 1, 2, and 4. Oh, and there was a 3, but that's now "old"?

Reply:
For some reason someone decided to start to count with 1, that's why we default to 1 if no version at all is set.